### ISSUE TITLE
can't connect even with correct credentials

### ISSUE THREAD

**Issue by All3xJ** [instabot-py/instabot.py#1149]
> Fill in all requirements *before* submitting an issue.
> Do not delete any of the template, fill all of it in; even if you think it doesn't apply to your issue.
> If you fail to follow these simple instructions, we will close the ticket.

- [x] This is a checked box. **Do not leave spaces around the `x`!**

- [x] I have searched for other issues with the same problem.

## Hosting
- [ ] Are you hosting the bot on a VPS or other Cloud hosting?
- [ ] If so; have you tried running the code on your local machine?

#### Select one:
- [x] Bug
- [ ] Feature Request
- [ ] Technical Help
- [ ] Other

### Operating System? (include version)
Linux

## Python Version?
2

## Description

Even if the credentials are correct, the tool says it can't log in. I tried also different accounts all with correct credentials, but nothing. Then the tool says (obviously) it can't like/comment/follow any account.

```
Include the contents of the error.log here (if applicable).
```

**Comment by noisyninjafeet** [comment]
I am not going to be of any help to you but I ran the bot for the very first time yesterday and it was working great as configured and ran all day. Today I changed a few parameters and it can now no longer log in. I suspect this is do to a change on instagrams side as I immediately get the "suspicious log in attempt is this you?" on my mobile device when trying to  run the bot. My 2 factor authentication is off.
